### DISCUSSION TITLE
Time-Scale Modification Algos / Speed up audio

### DISCUSSION THREAD

**Discussion by buczekEngineering** [coqui-ai/TTS#1850] [General]
Hello Guys,

The speech I recorded for finetuning the model, turns out to be a little bit to slow.
I was wondering, do you have any algorithms/methods to programatically speed up the audio.(without affecting the pitch)
I used wsola algo from pytsmod https://github.com/KAIST-MACLab/PyTSMod. However it makes the audio quality worse.
Do you have any methods?

**Comment by erogol** [comment]
There is no tool in 🐸TTS unfortunately for that.

**Reply by buczekEngineering** [reply]
Thanks for the response @erogol  Do you know any external libraries/projects that can handle audio manipulation in a post processing?
Implementing SSML tags that could handle various speech style adjustments would be quite helpful in general. This is the topic I'm currently investigating, but is hard to find anything.

**Reply by lexkoro** [reply]
You can try using ffmpeg to increase the speed

ffmpeg -i input.wav-filter:a "atempo=1.1" -vn output.wav

it shouldn't affect the pitch

**Reply by buczekEngineering** [reply]
Thanks a lot @lexkoro. It workes very well, and indeed does not affect the pitch.
